lib/containers/urls.pm

```diff
@@ -156,8 +156,7 @@ sub get_3rd_party_images {
     my @images = (
         "registry.opensuse.org/opensuse/leap",
         "registry.opensuse.org/opensuse/tumbleweed",
-        "public.ecr.aws/docker/library/alpine",
-        "public.ecr.aws/debian/debian",
+        "ghcr.io/linuxcontainers/alpine:latest"
```

Comment:
isn't there any download limit enforced by gh?

Reply:
Yes but we're only downloading only one image.

Reply:
multiplied by the archs and jobs

Reply:
> multiplied by the archs and jobs

Shall we remove this one as well?

Reply:
> multiplied by the archs and jobs

Let's merge and remove this one if and when we have issues. Merging to unblock o3.